apollo-ast/src/main/kotlin/com/apollographql/apollo3/ast/internal/ExecutableValidationScope.kt

```diff
@@ -106,7 +106,7 @@ internal class ExecutableValidationScope(
     name.forEach {
       builder.append(if (!isDecapitalized && it.isLetter()) {
         isDecapitalized = true
-        it.toString().toLowerCase(Locale.US)
+        it.toString().lowercase()
```

Comment:
That's the new Kotlin 1.5 lowercase() that we should be able to use now that we shadow the whole `stdlib` 🤞